qdrant-landing/content/blog/qdrant-1.8.x.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "Welcome to Qdrant 1.8.0!"
+draft: false
+slug: qdrant-1.8.x 
+short_description: "Look at what's new in Qdrant 1.8.0!"
+description: "Sparse vector performance, Text index loading optimization, Text immutability, Dynamic CPU saturation" 
+preview_image: /blog/qdrant-1.8.x/qdrant-1.8.0.png
+title_preview_image: /blog/qdrant-1.8.x/qdrant-1.8.0.png
+small_preview_image: /blog/qdrant-1.8.x/qdrant-1.8.0.png
+date: 2024-02-26T11:12:00-08:00
+author: Mike Jang
+featured: false 
+tags:
+  - vector search
+  - new features
+  - sparse vector performance
+  - dymamic CPU saturation
+  - text immutability
+  - text loading optimization
+weight: 0 # Change this weight to change order of posts
+---
+
+Today, we are pleased to announce the release of [Qdrant 1.8.0](https://github.com/qdrant/qdrant/releases/tag/v1.8.0).
+We've optimized performance in a number of ways. We've addressed issues with:
+
+<!-- Shard deltas (deferred to v1.9) -->
+- Sparse vectors
+- Dynamic CPU saturation
+- Text immutability
+- Text loading optimization
+
+We've also added <!-- TBD -->.
+
+Did we miss something? We rely on your feedback to drive our development. We
+welcome your contributions, especially in our [Discord community](https://qdrant.to/discord). Join us, introduce yourself, and help us build the best vector search engine!
+
+## Improved sparse vector performance
+
+The primary focus of Qdrant is performance. It's why we build our work in Rust.
+We also want to optimze performance in "real-world" situations.
+
+We used the [NeurIPS 2023 datasets](https://big-ann-benchmarks.com/neurips23.html). We've optimized our search to improve throughput by a factor of 16.
```

Comment:
The style is a bit heavy here IMHO, having three short sentences starting with `we`.

Reply:
I actually had 4 in a row.... :laughing: